NSS-Day-Cohort-69/tabloid-hamburgers
# Ft/post header

authors can upload post headers now instead of finding a URL for one online.

Changed post post and edit post endpoints to accept a form and image property as a file
Added image property to post model and post DTOs
Added a field for uploading a file in post form, and returns form as form data
Edit form component adds author id to form data
Post list and post details show image

To Test:

1. pull and checkout branch, restart API
2. add migration and update database, nuke database if that doesn't work
3. create a post, upload an image for the header
4. find post in post list, observe header image
5. go into post details, observe header image
6. edit a post, add an image
7. return to post list and observer image change

closes #10
